tasks/custom_task/custom_task.py

```diff
@@ -31,6 +31,21 @@ def get_dd_invoke_logs_path() -> str:
     return os.path.join(temp_folder, DD_INVOKE_LOGS_FILE)
 
 
+def get_running_mode() -> str:
```

Comment:
Discussed offline - we should return a list and have a field called `running_mode` instead that can have several values

Reply:
Done in [5776e64](https://github.com/DataDog/datadog-agent/pull/25860/commits/5776e644ae53d0598b5db0f6de4a17807ea14c7a). I made it a dict just like APIs